temporalcli/commands.operator_namespace.go

```diff
@@ -14,8 +14,25 @@ import (
 	"google.golang.org/protobuf/types/known/durationpb"
 )
 
+func (c *TemporalOperatorCommand) getNSFromFlagOrArg0(cctx *CommandContext, args []string) (string, error) {
+	if len(args) > 0 && c.Namespace != "default" {
+		return "", fmt.Errorf("namespace was provided as both an argument (%s) and a flag (-n %s); please specify namespace only with -n", args[0], c.Namespace)
+	}
+
+	if len(args) > 0 {
+		cctx.Logger.Warn("Passing the namespace as an argument is now deprecated; please switch to using -n instead.\n")
```

Comment:
Should not assume use of alias IMO

Reply:
If you feel strongly about this, I can reword (I suspect most people will use the short-form though).

Reply:
I think our docs should always show long form. We used to go overboard with aliases where you could even have the command aliased, so you could `tctl w l`. People fought to keep that lack of clarity too. Even with the aliases, I hope our documentation (and error messages) don't require alias lookup.